### DISCUSSION TITLE
Easy multitasking using tabs

### DISCUSSION THREAD

**Discussion by damionx7** [damionx7/Zettel-Notes-Documentation#184] [Feature Requests]
[apoc]: Here's an idea that would make Zettel unique and offer something that no other app does.  I've tried many note apps, the vast majority of which are here https://www.noteapps.ca/ but I don't know of any app for Android where it's easy to multitask or work on more than one note at the same time, it's like every developer thought the same, that a person only needs to write new information, there are no easy solutions for those who need to transfer information from one or several notes to another or consult one note while writing another.  My idea was for the app to have tabs like browsers, where in each tab it was possible to open a different note and switch between tabs to consult or edit the notes.  The tabs would appear at the bottom of the top area, the existing icons would be moved a little higher to create space for the tabs, they would have the same behavior as in a browser, when starting the app the contents of the initial window would already appear in a tab, next to the tab there would be a + sign to create a new tab, the content of each new tab was always the initial window. Zettel would not be a note-taking app among many others, but an app that stood out for being the only one capable of easy multitasking.

**Comment by damionx7** [comment]
[Sergio]: Wow, this functionality would take Zettel Notes to another level.
The proposed option of a button, as in Web browsers, has the drawback that it would take two clicks to reach a tab, so if you are going to work a lot with more than one file at a time, it was worth sacrificing a little screen.
Something practical would be for the tabs to only appear when you have more than one file open. I also think that the height of the tabs should be the minimum, showing the text of the file name they correspond to with a small letter.

**Comment by damionx7** [comment]
[Thomas Thai]: The option to simultaneously edit notes at the same time would help some people. However, having multiple always displayed tabs would take up precious mobile screen real estate, which is why most mobile browsers use a small button in the title bar or navigate bar to open a pop-up showing open tabs. Exploring for possible solutions would be worth while.
![37cc4a8c1fef0ebf4a0f57191abd3cbe](https://github.com/user-attachments/assets/87fb57ff-9613-4f2c-8e28-bb01bdffadca)
![184101fd40bb924d3dc63a32e272a6a4](https://github.com/user-attachments/assets/ee7c2476-4227-4cbc-9bd9-60198341662f)

**Comment by irmod** [comment]
I suggest creating four Tabs on the left Navigation Drawer.

First Tab for **Global** navigation and management:

- Calendar
- Gallery
- Repositories
- Attributes
- Plugins
- Recycle Bin

Make this Tab the Default. Allow the user to set their Default Tab selection in Settings.

The second tab can display **All Tags**. This tab will allow you to add tag management (rename, set the color for a tag, etc.). Sorting must be available for tags. Tags must be searchable. Below I have added a video with an example.

The third tab to manage **Saved Searches**.

On the fourth tab (let's call it "**Shelf**"), place a list of open documents (notes) for quick navigation through them. This may be similar to vertical tabs in a browser.

![Vertical_tabs_01.png](https://github.com/user-attachments/assets/bd587e7e-3909-4d5d-88dd-150bd5c46750)

Follow this link for a video of how tag search works and a general example of tabs

https://github.com/user-attachments/assets/4dc62b5d-788a-443b-a38d-36b3cc3824e4

**Comment by nathanl** [comment]
Some way of working with multiple notes at a time - for example, to cut and paste content from one to the other - would be great.

**Comment by nonplayer** [comment]
Tabs would be a godsend for me. I'm not concerned about screen real estate, especially at the top. Allow the bar to be hidden when only one document is open. Xoxo pdf reader does this pretty well, for an example.
